osu.Server.Queues.ScoreStatisticsProcessor/Stores/BuildStore.cs

```diff
@@ -15,34 +15,27 @@ namespace osu.Server.Queues.ScoreStatisticsProcessor.Stores
     /// </summary>
     public class BuildStore
     {
-        private readonly IReadOnlyDictionary<int, Build> builds;
+        private static readonly uint memory_cache_size_limit = uint.Parse(Environment.GetEnvironmentVariable("MEMORY_CACHE_SIZE_LIMIT") ?? "128000000");
```

Comment:
Probably not gonna be an issue due to the sliding expiration & overall expected low number of items here but I sort of wonder about this, three memory caches disparately read `MEMORY_CACHE_SIZE_LIMIT` for their own size limit, so I'm pretty sure the limit is going to be per cache rather than per the full process, which is maybe fine but I don't know if this was the intention